torch/_dynamo/variables/nn_module.py

```diff
@@ -131,18 +131,18 @@ class NNModuleVariable(VariableTracker):
     _nonvar_fields = {
         "module_type",
         "module_key",
-        "module",
+        "value",
         "nn_module_stack_source",
         *VariableTracker._nonvar_fields,
     }
 
     def __init__(
-        self, module_type: type, module_key: str, module: torch.nn.Module, **kwargs
+        self, module_type: type, module_key: str, value: torch.nn.Module, **kwargs
     ) -> None:
         super().__init__(**kwargs)
         self.module_type = module_type
         self.module_key = module_key
-        self.module = module
+        self.value = value
```

Comment:
Simple rename to make this compatible with the new `elif hasattr(arg, "value") and isinstance(arg.value, MutableMapping):` branch in `BuiltinVariable.call_custom_dict`.